cameraViewEx/src/main/api21/com/priyankvasa/android/cameraviewex/Camera2.kt

```diff
@@ -1250,6 +1253,7 @@ internal open class Camera2(
     }
 
     override fun stopVideoRecording(): Boolean = runCatching {
+        listener.onVideoRecordStopped()
```

Comment:
`onVideoRecordStopped()` should be called after `mediaRecorder?.stop()` to make sure it is only called if recording was actually stopped.

Reply:
Seems ok to me, moved.